lightning/src/routing/scoring.rs

```diff
@@ -474,7 +474,84 @@ where L::Target: Logger {
 	decay_params: ProbabilisticScoringDecayParameters,
 	network_graph: G,
 	logger: L,
-	channel_liquidities: HashMap<u64, ChannelLiquidity>,
+	channel_liquidities: ChannelLiquidities,
+}
+/// ChannelLiquidities contains live and historical liquidity bounds for each channel.
```

Comment:
Objections to moving this into its own file?

Reply:
For my workflow, sticking to one struct per file would work well. I do find myself navigating quite a bit in these large files, using editor features (find, find symbol, find ref) to make it easier but not perfect. I'd rather use the folder/file hierarchy and pinning of files as tabs for example. But it is personal of course.

Some type of split would be welcome either way. For this PR I could start with a `liquidity_information` (open to naming suggestions) module and place the new `ChannelLiquidities` in it. Then in a separate PR move more liquidity code (`ChannelLiquidity`, `HistoricalLiquidityTracker`, `HistoricalBucketRangeTracker` and tests) in there. Thoughts?

Reply:
Skipping the moves for now as this PR is getting close to ready.